apph-back/src/main/java/com/viseo/apph/dao/S3Dao.java

```diff
@@ -36,7 +30,7 @@ public class S3Dao {
     @Autowired
     S3Client s3Client;
 
-    private String hashString(String string) {
+    public String hashString(String string) {

```

Comment:
```suggestion
    private String hashString(String string) {
```

cf : le commentaire dans PhotoRequest.java